hoomd/md/NeighborList.cc

```diff
@@ -524,12 +518,22 @@ void NeighborList::checkBoxSize()
         rmax += 0.5*max_d_comp;
         }
 
+    std::cout << "rmax: " << rmax << std::endl;
```

Comment:
Was just skimming my emails, and this PR caught my eye. This looks like a stray debug output. Otherwise looks good!

Reply:
Thanks! Good catch.